### ISSUE TITLE
yii2 MongoDB extension is old

### ISSUE THREAD

**Issue by xiaoyi510** [yiisoft/yii2-mongodb#322]
MongoDB 4.x 已经支持事务,但是yii2扩展mongodb暂不支持事务操作

MongoDB 4.x already supports transactions, but yii2 extended mongodb does not support transaction operations temporarily

**Comment by samdark** [comment]
There are two pull requests:

- https://github.com/yiisoft/yii2-mongodb/pull/294
- https://github.com/yiisoft/yii2-mongodb/pull/276

In order to properly check it we need to switch to GitHub actions and fix tests: https://github.com/yiisoft/yii2-mongodb/issues/279

**Comment by alxark** [comment]
any plans to implement transactions?

**Comment by bizley** [comment]
Would you like to help with the implementation?

**Comment by alxark** [comment]
> Would you like to help with the implementation?

what about this PR https://github.com/yiisoft/yii2-mongodb/pull/276  ? at least to start from.

**Comment by bizley** [comment]
Yes, you can continue from there. We switched to Github actions so it should be easier, the PR requires tests though.
